docs/guides/typescript.md

```diff
@@ -117,6 +117,31 @@ const myMachineConfig: MachineConfig<TContext, TSchema, TEvent> = {
 };
 ```
 
+## Actions
+
+The `send` action on the interpreted machine `interpret(stateMachine)` isn't always type safe. To get typechecking for this function signature, use the following pattern:
+
+```ts
+type UserEvents =   {
+    type: "TEST";
+    value: string;
+}
+
+const service = interpret(stateMachine);
+
+// This will compile
+service.send({ type: "TEST", value: "testvalue" });
+
+// This will have an
```

Comment:
Is this missing a couple words?

Reply:
Yeah, wanted to say that it would generate an error. You've approved, though. Should I change?

Reply:
Yes please